google/cloud/forseti/scanner/scanner.py

```diff
@@ -92,33 +91,33 @@ def run(forseti_config, model_name=None, service_config=None):
 
     if forseti_config is None:
         LOGGER.error('Path to Forseti Security config needs to be specified.')
-        sys.exit()
+        return 1
 
     try:
         configs = file_loader.read_and_parse_file(forseti_config)
     except IOError:
         LOGGER.error('Unable to open Forseti Security config file. '
                      'Please check your path and filename and try again.')
-        sys.exit()
+        return 1
     global_configs = configs.get('global')
     scanner_configs = configs.get('scanner')
 
     log_util.set_logger_level_from_config(scanner_configs.get('loglevel'))
 
     # TODO: Figure out if we still need to get the latest model here,
     # or should it be set in the server context before calling the scanner.
-    snapshot_timestamp = _get_timestamp(global_configs)
-    if not snapshot_timestamp:
-        LOGGER.warn('No snapshot timestamp found. Exiting.')
-        sys.exit()
+    #snapshot_timestamp = _get_timestamp(global_configs)
+    #if not snapshot_timestamp:
+    #    LOGGER.warn('No snapshot timestamp found. Exiting.')
+    #    sys.exit()
 
     violation_access = scanner_dao.define_violation(
         model_name, service_config.engine)
     service_config.violation_access = violation_access
 
     runnable_scanners = scanner_builder.ScannerBuilder(
         global_configs, scanner_configs, service_config, model_name,
-        snapshot_timestamp).build()
+        model_name).build()
```

Comment:
This model_name should not be repeated.

Reply:
Yes, I tested the functionality -- the reason for this PR is that I was having trouble getting scanner to work on my side.

The snapshot timestamp is not used as far as I can tell, for the firewall scanner. However, because we've moved over one scanner, the other scanners still use the snapshot timestamp (it seems) and I didn't want to totally drop it.